geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/GMSMembership.java

```diff
@@ -1545,7 +1488,7 @@ private boolean isNew(final ID m) {
    * <p>
    * Like isShunned, this method holds the view lock while executing
    *
-   * Concurrency: protected by {@link #latestViewLock} ReentrantReadWriteLock
+   * Concurrency: protected by {@link #latestViewReadLock} ReentrantReadWriteLock
```

Comment:
Typo: it's a re-entrant read lock, not a re-entrant read-write lock

Reply:
I misspoke and the comment is still wrong. This would be better:

```java
* Concurrency: protected by {@link #latestViewReadLock}
```

no such thing as a "ReentrantReadWriteLock". Sorry for the confusion!